src/event-bus.js

```diff
@@ -0,0 +1,4 @@
+import Vue from 'vue';
```

Comment:
A global event bus. I googled so much on how to receive events from parent to child since `broadcast` was deprecated. There was so much on child to parent but not the other way around. Perhaps I'm missing something? This was the solution I arrived at.